Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/ImageExBase.Members.cs

```diff
@@ -139,5 +150,14 @@ public ImageExCachingStrategy CachingStrategy
             get { return (ImageExCachingStrategy)GetValue(CachingStrategyProperty); }
             set { SetValue(CachingStrategyProperty, value); }
         }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets is lazy loading enable. (17763 or higher supported)
+        /// </summary>
+        public bool EnableLazyLoading
```

Comment:
Add Remarks here for version info `<remarks>Windows 10 build 17763 or higher required.</remarks>`